Neos.Flow/Classes/Error/Debugger.php

```diff
@@ -467,7 +467,10 @@ public static function getCodeSnippet(string $filePathAndName, int $lineNumber,
     protected static function getObjectSnippetPlaintext(object $object): string
     {
         if (is_callable([$object, '__toString'])) {
```

Comment:
maybe `method_exist` but yeah i get the point while rendering an exception trace there are more important manners than to find out that the __toString implementation of some stupid object doesnt work. lIke our eel context which hasnt worked for ages ...

Reply:
Ah mmm yes maybe we change that anyways...